core/src/main/java/org/apache/accumulo/core/util/compaction/RunningCompaction.java

```diff
@@ -66,4 +74,12 @@ public String getGroupName() {
     return groupName;
   }
 
+  public Long getStartTime() {
+    return startTime;
+  }
+
+  public void setStartTime(Long time) {
+    startTime = time;
+  }
```

Comment:
For these two method would make getStartTime() throw an exception if its not set and setStartTime() throw an exception if it is set (or it could ignore subsequent calls to set after the first, not sure what is best).  Recommending this because these values are being used for a key in a set and if they change that will cause problems for the set.